catalog-info.yaml

```
@@ -8,6 +8,6 @@ metadata:
   description: "A django application to provide access to search services"
   links: []
 spec:
-  owner: group:openedx-unmaintained
+  owner: Ali-Salman29
```

```suggestion
  owner: "user:Ali-Salman29"
```

Usernames should be prefixed via `user:` per https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.html#references